app/models/league.rb

```diff
@@ -1,6 +1,8 @@
 class League < ActiveRecord::Base
   BASE_SCORE = 1_200
 
+  mount_uploader :league_header_image, LeagueHeaderImageUploader
```

Comment:
note on naming (you don't _have_ to change this, but it'd be nice): why is it called `league_header_image` when it's an attribute of a league? we don't call it `@article.article_title` either, do we?
`LeagueHeaderImageUploader` is fine, there you need the prefix to distinguish it from uploaders for other header images. no need to have the same prefix on the league attribute, though, if you ask me.